### DISCUSSION TITLE
getting error while starting  Valheim

### DISCUSSION THREAD

**Discussion by MidhunVijayan** [GameServerManagers/LinuxGSM#3265] [Q&A]
hi ,

i am getting the following error while starting valheim 

```

no boot config - using default values
(Filename:  Line: 476)

```

can you please help me on this?

thank you!

**Comment by h3o66** [comment]
Do you have any more details ?
Basic infos like:
Distro (+ Version)
What you did set in the lgsm configs ?

**Comment by MidhunVijayan** [comment]
hi,

I renamed valheim_server.x86_64 before starting that's why i got the error as above.

i have used the same name valheim_server.x86_64 now, and currently getting the error as below

```
dlopen failed trying to load:
steamclient.so
with error:
steamclient.so: wrong ELF class: ELFCLASS32
[S_API FAIL] SteamAPI_Init(): Sys_LoadModule failed to load: steamclient.so
[S_API FAIL] SteamAPI_Init() failed
[Steamworks.NET] GameServer.Init() failed.
(Filename: ./Runtime/Export/Debug/Debug.bindings.h Line: 35)
```
i am using debian 10.8

**Reply by wldwalker** [reply]
Yeah, try not to rename anything. You should just be using the LGSM config files if you need to change anything. Were you able to get your servers running?
